assets/src/blocks/Gallery/GalleryBlock.js

```diff
@@ -1,207 +1,96 @@
-         *                    gallery_block_focus_points="{'23596':'left top','23597':'left top','23599':'left top'}"
-         * /]
-         *
-         * new block-gutenberg:
-         * <!-- wp:planet4-blocks/gallery {"gallery_block_style":3,"gallery_block_title":"test title","gallery_block_description":"test desc",
-         *      "multiple_image":"23603,23602,23600,23596"} /-->
-         *
-         */
-        transforms: {
-          from: [
-            {
-              type: 'shortcode',
-              // Shortcode tag can also be an array of shortcode aliases
-              tag: 'shortcake_gallery',
-              attributes: {
-                gallery_block_style: {
-                  type: 'integer',
-                  shortcode: ({named: {gallery_block_style = ''}}) => Number(gallery_block_style),
-                },
-                gallery_block_title: {
-                  type: 'string',
-                  shortcode: ({named: {gallery_block_title = ''}}) => gallery_block_title,
-                },
-                gallery_block_description: {
-                  type: 'string',
-                  shortcode: ({named: {gallery_block_description = ''}}) => gallery_block_description,
-                },
-                multiple_image: {
-                  type: 'string',
-                  shortcode: ({named: {multiple_image = ''}}) => multiple_image,
-                },
-                gallery_block_focus_points: {
-                  type: 'string',
-                  shortcode: ({named: {gallery_block_focus_points = ''}}) => gallery_block_focus_points,
-                },
-              },
-            },
-          ]
+    registerBlockType(BLOCK_NAME, {
+      title: __('Gallery', 'planet4-blocks-backend'),
+      icon: 'format-gallery',
+      category: 'planet4-blocks',
+      attributes,
+      supports: {
+        html: false, // Disable "Edit as HTMl" block option.
+      },
+      edit: GalleryEditor,
+      save: frontendRendered(BLOCK_NAME),
+      // Add our custom styles
+      styles: [
+        {
+          name: 'slider',
+          label: getStyleLabel(
+            'Slider',
+            'The slider is a carousel of images. For more than 5 images, consider using a grid.'
+          ),
+          isDefault: true
         },
-        attributes: {
-          gallery_block_style: {
-            type: 'number',
-            default: 1
-          },
-          gallery_block_title: {
-            type: 'string',
-          },
-          gallery_block_description: {
-            type: 'string',
-          },
-          multiple_image: {
-            type: 'string',
-          },
-          gallery_block_focus_points: {
-            type: 'string',
-          },
-          image_data: {
-            type: 'array',
-            default: []
-          },
+        {
+          name: 'three-columns',
+          label: getStyleLabel(
+            '3 Column',
```

Comment:
Column**s** ? (same next line)

Reply:
I've kept the old copy here but I think you're right indeed, there should be an "s" 😄